src/App.jsx

```diff
@@ -4,19 +4,36 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 import { Nav } from "./components";
 import ArticlePage from "./components/ArticlePage";
 import { Stack } from "@mui/material";
+import { useMediaQuery } from "@mui/material";
+import { useMemo } from "react";
+import { createTheme } from "@mui/material";
+import { ThemeProvider } from "@mui/material";
 
 function App() {
+  const prefersDarkMode = useMediaQuery("(prefers-color-scheme: dark)");
+  const theme = useMemo(
+    () =>
+      createTheme({
+        palette: {
+          mode: prefersDarkMode ? "dark" : "light",
```

Comment:
Nice, as I mentioned this wasn't super urgent but I love that you tackled it! Light mode for me still has a black background but the articles are white, so it's almost there :D